### ISSUE TITLE
Support for Regression Problems

### ISSUE THREAD

**Issue by cancan101** [NervanaSystems/neon#33]
Where the output is real values `(R^n)` and the loss function is something like MSE.

**Comment by apark263** [comment]
You can use the sum_squared cost (see neon/transforms/sum_squared.py)

We'll try to provide an example at some point, but it should be as simple
as swapping the CrossEntropy item in the CostLayer to be SumSquaredDiffs
instead
